core/imt/test/testEnableImt.cxx

```diff
@@ -0,0 +1,26 @@
+#include "TROOT.h"
+#include "ROOT/RTaskArena.hxx"
+
+#include "tbb/task_arena.h"
+
+#include "ROOT/TestSupport.hxx"
+#include "gtest/gtest.h"
+
+#ifdef R__USE_IMT
+
+static const unsigned gMaxConcurrency = ROOT::Internal::LogicalCPUBandwidthControl();
+
+TEST(EnableImt, TBBAttach)
+{
+   tbb::task_arena main_arena;
```

Comment:
Why not just
```suggestion
   tbb::task_arena main_arena(2);
```
?

Reply:
I used the pattern used internal in ROOT.  I am agnostic :)